plugins/generator-1.20.4/neoforge-1.20.4/templates/elementinits/items.java.ftl

```diff
@@ -105,7 +124,7 @@ public class ${JavaModName}Items {
 
 	<#if hasItemsWithProperties>
 	<#compress>
-	@SubscribeEvent public static void clientLoad(FMLClientSetupEvent event) {
+	@SubscribeEvent @OnlyIn(Dist.CLIENT) public static void clientLoad(FMLClientSetupEvent event) {

```

Comment:
If this is a bug, please also make a PR backporting this fix

Reply:
Before this side marker was added to the class annotation. If capabilities are client-side only as well, I can move it back.

Reply:
Ah, I see. But does this work with dedicated server, as the class containing method with this signature will still be loaded?

Please test with dedicated server (not LAN) and item having both inventory and item states to make sure it does not crash

Reply:
Does your demo workspace cover this case? In this case I can try to test

Reply:
Yes, I first enabled all parameters and reverted them except exactly inventory and states for later tests.

Reply:
I tested, it still works and doesn't crash the game.